### PR TITLE
Feature flags larger pages and rename sidebar item

### PR DESCRIPTION
**Pull request by timgl**
## Changes

- Rename "Experiments" to "Feature Flags" as I think this might be confusing people. We will have experiments at some point but for now it's just feature flags 
image
- Show way more feature flags per page

## Checklist

- [ ] All querysets/queries filter by Team (if this PR affects any querysets/queries)
- [ ] Backend tests (if this PR affects the backend)
- [ ] Cypress E2E tests (if this PR affects the front and/or backend)

### CODE REVIEW

File: frontend/src/layout/Sidebar.js [hunk 1]
```diff
@@ -228,7 +228,7 @@ export function Sidebar({ user, sidebarCollapsed, setSidebarCollapsed }) {
                     </Menu.SubMenu>
                     <Menu.Item key="experiments" style={itemStyle} data-attr="menu-item-feature-f">
                         <ExperimentOutlined />
```
Comment (yakkomajuri) [inline frontend/src/layout/Sidebar.js:230]: We should maybe then consider switching up this icon? Although Antd's flag icons are pretty ugly

**Review by yakkomajuri — APPROVED** [review]
Agree with the change. Would consider switching up the icon.

File: frontend/src/layout/Sidebar.js (hunk 1 above)
Comment (yakkomajuri) [reply frontend/src/layout/Sidebar.js:230]: Maybe ForkOutlined? @paolodamico thoughts?

Comment (Twixes) [reply frontend/src/layout/Sidebar.js:230]: There's actually `FlagOutlined`, which could make sense?

Comment (yakkomajuri) [reply frontend/src/layout/Sidebar.js:230]: > Although Antd's flag icons are pretty ugly

That's my problem with it

Comment (paolodamico) [reply frontend/src/layout/Sidebar.js]: Depending on #1775 we might start introducing some custom icons for some stuff where default stuff isn't enough (e.g. retention, feature flags, ...), so I wouldn't worry to much about this right now.

**Pull request merged**